CONTRIBUTING.rst

```diff
@@ -664,6 +664,63 @@ Later on
+
+* The devlist for official communication, discussing proposals, voting, general issues, asking community for
+  opinion etc.
+* Wiki for detailed discussions on big proposals (AIPs) and some helpful, shared resources.
+* PRs for discussing implementation details of this PR (not for architectural discussions)
+* Slack for ad-hoc questions, troubleshooting, occassional discussions, group talks (including SIG - special
+interest groups), notifications, random queries, newbie questions etc.
+
+
+It is important to remember that all contributors, committers and even PMC members are individuals
+that contribute a lot of their free time on top of their work time to Apache Airflow.
+They have families, personal life, sometimes they are swamped with work, sometimes they are on vacations
+sometimes they are simply tired and need a time off. They are in various timezones and their responsiveness
+will vary from time to time. As a community project we have no formal structure, we have no managers nor
+departments and most of the people are autonomous in their opinions and decisions, so sometimes even
+committers will have different opinions on the same subjects.
+
+Disagreements are expected, discussions might include strong opinions and contradicting statements.
+Sometimes you might get two committers asking you to do things differently. This all happened in the past
+and will continue to happen. As a community we have some mechanisms to facilitate discussion and come to
+a consensus, conclusions and sometime we end up voting certain decisions. It is important that those
+decisions are not treated as personal wins or looses - at the end it's the community that we care about
+and what's good for community, should be accepted even if you have different opinion. There is the nice
+motto that you should follow in case you disagree with community decision "Disagree but engage". Even
+if you do not agree with a community decision, you should follow it and embrace (but you are free to
+express your opinion that you don't agree with it).
+
+Here are a few rules that are important to keep in mind when you enter our community:
+
+ * Do not be afraid to ask questions
+ * We have deeply asynchronous communication - do not expect immediate answers, ping people if you are blocked
+ * We have #newbie-questions channel in slack as a safe place to ask questions
+ * You can ask one of the committers to be a mentor for you, they can guide within the community
+ * You can apply to more structured Apache Mentoring Programme
+ * It’s your responsibility as an author to take your PR from start-to-end including leading communication
+   in the PR
+ * It’s your responsibility as an author to ping committers to review your PR - be mildly annoying sometimes,
+   it’s OK to be slightly annoying with your change - it is also a sign for committers that you care
+ * We have high-bar set for the code requirements - be considerate to those requirements
```

Comment:
Hm, maybe a short note why they are so high?

Reply:
Added it as a chapter above